Remove extra whitespace at the end of recipients string

AuroreM opened the pull request:
Hi @tkporter 

As discussed in [here](https://github.com/tkporter/react-native-sms/issues/41) I worked on the issue of the "whitespace contact" automatically added on my phone.
I manage to fix it by removing the last character of the recipientString which is always a whitespace.

![whatsapp image 2017-12-22 at 13 51 20](https://user-images.githubusercontent.com/15011364/34298798-cc7dc3c0-e71f-11e7-85a6-c4def9bc08d3.jpeg)

I'm not familiar with Java (I'm only coding in RN), don't hesitate to make some comments :)

tkporter commented:
Thanks for checking this out! Could you try changing the separator variables (lines 84 and 86) to have no spaces and see if that has the same effect instead of using substring?

AuroreM commented:
@tkporter I've removed the whitespace after the coma and the semi-colon and it works.

As it's a better solution than the substring, I've updated the PR :)

tkporter commented:
Awesome, thank you!
